### DISCUSSION TITLE
Adding pyupgrade to .pre-commit-config.yml

### DISCUSSION THREAD

**Discussion by kulbachcedric** [online-ml/river#1083] [Ideas]
Hi,
what do you think about adding [`pyupgrade`](https://github.com/asottile/pyupgrade) to the `.pre-commit-config.yml`?
```
      - id: pyupgrade
        name: pyupgrade
        language: python
        types: [ python ]
        entry: pyupgrade --py38-plus
```
However, when running `pyupgrade` on all `.py` files e.g. `pyupgrade "find . -name "*.py"` there are quite some changes within the repository.

**Comment by MaxHalford** [comment]
Hey, I guess it doesn't hurt. Some of it looks redundant with black, but I do see the benefits of it. I don't see any reason to say no, except that it's yet another (dev) dependency.
